docs/self-managed/setup/deploy/openshift/redhat-openshift.md

```diff
@@ -326,37 +326,6 @@ tasklist:
 The actual configuration properties can be reviewed [in the Tasklist configuration documentation](/self-managed/tasklist-deployment/tasklist-configuration.md#zeebe-broker-connection).
 
 5. For Connectors:
```

Comment:
Can this be merged into step 4 now that Connectors don't have a separate set of requirements/steps?

Reply:
@conceptualshark Connectors have a different set of env vars. I think it is still a good idea to have them separate.